### DISCUSSION TITLE
إمكانية بناء نسخة للأجهزة القديمة ؟

### DISCUSSION THREAD

**Discussion by Al-fata-tayeb** [rn0x/Altaqwaa-Islamic-Desktop-Application#55] [Q&A] [answered]
هل يمكن بناء نسخة لأجهزة i386 المعروفة كذلك بإسم i686 و 86x المعروفة بإسم 32bit architecture

**Comment by kemzops** [comment, marked as answer]
للأسف بناء التطبيق بمعمارية 32 بيت لا ينصح فيه ابدا في الوقت الحالي بسبب عدم دعم بعض التبعيات الجديدة لاليكترون له وخصوصا على لينكس شبه مستحيل ولن يعمل مثل نسخة 64 بيت... @Al-fata-tayeb (بعكس لو ويندوز 32 بيت **من الممكن** ان يعمل بنفس الوظائف) .

وحتى تشغيل البرنامج على نظام تشغيل **ويندوز 7** او اقل ايضا قريبا سيصبح غير مدعوم لاستخدام البرنامج لـ **`electronjs`.**

يوجد بعض الاقتراحات للايجاد حل او بديل... ولكن كلها تعتمد عليك.
